src/lib/npm.ts

```diff
@@ -168,6 +161,7 @@ export const ensureNpmDependency = async ({
   });
 
   if (
+    global === false &&
     maybeFluenceConfig !== null &&
     version !==
       (maybeFluenceConfig?.dependencies?.npm?.[name] ??
```

Comment:
This condition is something :D

Can you simplify it? Move calculations to separate variables, add comments, etc

Reply:
agree. This is completely unredable. Will improve